policyengine_us/parameters/gov/states/ky/tax/income/deductions/itemized.yaml

```diff
@@ -0,0 +1,16 @@
+description: Kentucky allows these itemized deductions.
+values:
+  2022-01-01:
+    - home_mortgage_interest
+    - investment_income_form_4952
+    - charitable_cash_donations
+    - charitable_non_cash_donations
+    - gambling_losses_capped_at_winnings
```

Comment:
turns out this is a federal deduction with the same logic. could you add it to the list of federal itemized deductions? then it'll flow through if we omit it from excluded deductions per below